java/src/main/java/sh/platform/languages/App.java

```diff
@@ -0,0 +1,9 @@
+package sh.platform.languages;
+
+public class App {
+
+    public static void main(String[] args) throws Exception {
+        JettyServer jettyServer = new JettyServer();
+        jettyServer.start();
```

Comment:
I am entirely ignorant here, but isn't Jetty pretty heavy?  I thought SpringBoot was the lightweight option and Jetty was on the Tomcat level.

Reply:
Spring Boot needs a server container such a Jetty and Tomcat.
This project has more infrastructure code than a business; that's why I do not put Spring as one more layer.
It is a peculiar Java code.

https://www.dailyrazor.com/blog/tomcat-vs-jetty/